notebooks/266-table-question-answering/README.md

```diff
@@ -0,0 +1,32 @@
+# TAPAS base model fine-tuned on WikiTable Questions (WTQ) with OpenVINO
+
```

Comment:
if it works in colab / binder, please add links

Reply:
Added for colab